client/docs/py-primer-dark.html

```diff
@@ -0,0 +1,178 @@
+<html>
+  <head>
+    <style>
+      a {
+        color: #82B1FF;
+      }
+      body {
+        background-color: #333a42;
+        color: #e0e0e0;
+        font-family: Helvetica, Arial;
+        font-size: 14px;
+      }
+      p {
+        width: 616px;
+      }
+      pre {
+        color: white;
+        font-size: 13px;
+        padding-top: 8px;
+        padding-bottom: 8px;
+        width: 632px;
+      }
+      .function-name {
+        color: #e9eded;
+      }
+      .general-statements {
+        color: #c792ea;
+      }
+      .h1 {
+        color: white;
+        font-size: 24px;
+        height: 50px;
+        line-height: 50px;
+        text-align-last: center;
+        width: 616px;
+      }
+      .h2 {
+        background-color: #4242b7;
+        color: #e0e0e0;
+        font-size: 16px;
+        height:32px;
+        line-height: 32px;
+        margin-bottom: 24px;
+        margin-left: -8px;
+        margin-right: -8px;
+        margin-top: 24px;
+        text-align-last: center;
+        width: 640px;
+      }
+      .method-type {
+        color: #decb6b;
+      }
+      .names {
+        color: #6f96e0;
+      }
+      .number {
+        color: #f77669;
+      }
+      .property {
+        color: #669c87;
+      }
+      .single-line-comment {
+        color: #7599a9;
+      }
+      .string {
+        color: #a0b97c;
+      }
+      .transparent {
+        font-size: 0px;
+        height: 0px;
+        opacity: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="h1">
+      <span class="transparent">"""</span>
+      PYTHON IN 2 MINUTES (OR SO)
+      <span class="transparent">"""</span>
+    </div>
+    <p>
+      <span class="transparent">"""</span>The following is a very short primer
```

Comment:
Not sure I understand the class="transparent" thing. Can a similar effect be achieved with CSS rather than using """ characters?

Reply:
This page supports full page copy/paste onto Python interpreter. The section headings, therefore, need to be commented out with """, but stylistically, I don't want the section headers or the footer to show """. If you do a select all on the page and paste onto a test file, you'll see the headers and footer are commented out. It's one less thing for the user to get tripped up on if they decide to copy/paste from this page.

Reply:
Ah. If that's the motivation, I recommend not optimizing for that. There's pedagogical value in typing stuff out, rather than copy/pasting.

If you want to keep it, though, consider adding an inline comment above the first use of the class (in both files) to explain the rationale, so that others understand why it's there too.

Reply:
Added comment below style section.